lib/Inkscape.js

```diff
@@ -1,9 +1,32 @@
 const childProcess = require('child_process');
+const semver = require('semver');
 const Stream = require('stream').Stream;
 const fs = require('fs');
+const util = require('util');
 const getTemporaryFilePath = require('gettemporaryfilepath');
 const async = require('async');
 
+const execFileAsync = util.promisify(childProcess.execFile);
+
+// check if the installed inkscape version is compatible
+let isCompatibleInkscape;
```

Comment:
This never gets set?

Would be nice to memoize the promise for `checkInkscape` so that we can guarantee that `inkscape --version` only gets executed once, even if a second `Inkscape` instance is created before the first one has completed the check.

Reply:
Ugh, yes it was supposed to be set - my thinking was once it’s a boolean it’s effectively memoized but yep that doesn’t fix the race 🤦‍♂️ Will fix it.